installer/templates/new/app.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Phoenix",
+  "description": "Elixir Web Framework targeting full-featured, fault tolerant applications with realtime functionality",
+  "website": "http://www.phoenixframework.org/",
+  <%= if ecto do %>"scripts": {
+    "postdeploy": "mix ecto.migrate"
+  },<% end %>
+  "env": {
+    "SECRET_KEY_BASE": {
```

Comment:
This automatically generates a random secret and adds it to Heroku as a SECRET_KEY_BASE. User doesn't need to set it. I added a `System.get_env("SECRET_KEY_BASE")` to prod.secret.exs. 

By the way, also tested this from this https://github.com/gjaldon/test. I also set it to private and added a template parameter and it worked fine too.